tests/x11/reboot_gnome.pm

```diff
@@ -16,11 +16,26 @@ use strict;
 use testapi;
 use utils;
 
+sub assert_and_click_until_screen_change {
+    my ($mustmatch, $wait_change, $repeat) = @_;
+    $wait_change //= 2;
+    $repeat      //= 3;
+    my $i = 0;
+
+    for (; $i < $repeat; $i++) {
+        wait_screen_change { assert_and_click $mustmatch; }, $wait_change;
```

Comment:
honestly I don't know why but this gives me `Useless use of private variable in void context at /var/lib/openqa/cache/lord.arch.suse.de/tests/sle/tests/x11/reboot_gnome.pm line 26.` in logfiles. I guess you can see this in other logfiles as well.